Server/Server.Exec/ArgParser.hpp

```diff
@@ -0,0 +1,130 @@
+#pragma once
+
+#include <argparse/argparse.hpp>
+#include <FileLogger.hpp>
+
+/**
+ * @class ArgParser.
+ * @brief This class parses arguments with which application is started.
+ * @details
+ * For example: "--port 5432 --dbport 65001 --dbname juniorgram --hostaddr 127.0.0.1 --user postgres --password postgres".
+ * Sending the port to the Server class constructor and list of all arguments to the PostgreAdapter Instance Method.
+ */
+
+class ArgParser
+{
+public:
+    /**
+     *@brief It's deleted default constructor. There is no need to use this.
+     */
+    ArgParser() = delete;
+   
+    /**
+     *@brief It's deleted copy constructor. There is no need to use this.
+     */
+    ArgParser(const ArgParser&) = delete;
+   
+    /**
+     * @brief Used to take parameters from main function.
+     * @details Parses arguments which were set when application was run.
+     * @param int argc(amount of arguments), const char** argv(C-string arguments).
+     */
+    ArgParser(int argc, const char** argv) 
+    {
+        _program = argparse::ArgumentParser("ServerArguments", "0.6", argparse::default_arguments::none);
```

Comment:
What is the "0.6"?

Reply:
Ve haven't any versioning tool, so for now it will be like that